blocks/story-carousel-block/features/story-carousel/default.jsx

```diff
@@ -0,0 +1,131 @@
+import React from "react";
+import PropTypes from "@arc-fusion/prop-types";
+import { Carousel, Image, Icon, Heading, Paragraph, Stack } from "@wpmedia/arc-themes-components";
+import { useContent } from "fusion:content";
+import { extractImageFromStory } from "@wpmedia/resizer-image-block";
+import { useFusionContext, useComponentContext } from "fusion:context";
+import getProperties from "fusion:properties";
+import getTranslatedPhrases from "fusion:intl";
+
+const BLOCK_CLASS_NAME = "b-story-carousel";
+
+const StoryCarousel = ({
+	customFields: {
+		inheritGlobalContent = false,
+		carouselContentConfig: { contentService = "", contentConfigValues = {} } = {},
+		headerText,
+		itemHeaderTruncationLines = 0,
+		itemDescriptionTruncationLines = 0,
+	},
+	globalContent = {},
+}) => {
+	const { id } = useComponentContext();
+	const { arcSite } = useFusionContext();
+	const { locale = "en" } = getProperties(arcSite);
+	const phrases = getTranslatedPhrases(locale);
+
+	const content = useContent({
```

Comment:
We should make use of the filter param here to ensure we are only getting the data we need, we also need to specify an additional query param to avoid caching issues when using `filter` as per this example - https://github.com/WPMedia/arc-themes-blocks/blob/arc-themes-release-version-1.21/blocks/small-promo-block/features/small-promo/default.jsx#L15

Reply:
filter has been added